src/util/system.cpp

```diff
@@ -74,9 +74,14 @@
 #include <malloc.h>
 #endif
 
+#include <univalue.h>
```

Comment:
20744 - partial due to missing changes in `src/util/syscall_sandbox.cpp` ?

Reply:
Bitcoin eventually does away with syscall sandboxing entirely (see [bitcoin#27896](https://github.com/bitcoin/bitcoin/pull/27896)), it's very likely any backports related to it will be marked DNM.